### PR TITLE
Impelented: logic to select location on pdp page(#22qm8pq)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2022-03-01 13-49-26

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)
- [ ] I have changed the target to hacktoberfest branch

### CODE REVIEW

File: src/views/count.vue [hunk 1]
```diff
@@ -97,10 +105,44 @@
...
     },
+    data(){
+      return{
+        facilityLocation: [] as any,
+        pickerOptions: {} as any,
+        locationId: ""
+      }
+    },
+    async mounted(){
+      this.facilityLocation = await this.store.dispatch("user/getFacilityLocation", {facilityId: this.facility.facilityId, productId: this.product.productId});
```
Comment (adityasharma7) [inline src/views/count.vue:120]: I think you should call service directly, we are not mutating the state, action is not needed

Comment (adityasharma7) [reply src/views/count.vue:120]: Also, improve variable name, I think we are getting `facilityLocations`

File: src/views/count.vue [hunk 2]
```diff
@@ -97,10 +105,44 @@
...
     methods: {
+      async openLocationPicker() {
+        this.pickerOptions.options = await this.facilityLocation.data.docs.map((location: any) => {
+          return { text:location.locationSeqId, value: location.locationSeqId };
+        })
+        const picker = await pickerController.create({
+          columns: [this.pickerOptions],
+          buttons: [
+            {
+              text: "Cancel",
```
Comment (adityasharma7) [inline src/views/count.vue:132]: This should be localised

File: src/views/count.vue [hunk 3]
```diff
@@ -97,10 +105,44 @@
...
+              role: "cancel",
+            },
+            {
+              text: "Confirm",
```
Comment (adityasharma7) [inline src/views/count.vue:136]: This should be localised

File: src/views/count.vue [hunk 4]
```diff
@@ -97,10 +105,44 @@
...
+              handler: (value) => {
+                this.locationId = value.undefined.value;
```
Comment (adityasharma7) [inline src/views/count.vue:138]: Why `value.undefined.value`?

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/count.vue (hunk 1 above)
Comment (meet-aniket) [reply src/views/count.vue:120]: Removed action from user module and written a method in **vue** file to fetch locations.

File: src/views/count.vue (hunk 2 above)
Comment (meet-aniket) [reply src/views/count.vue:132]: Text is localized.

File: src/views/count.vue (hunk 3 above)
Comment (meet-aniket) [reply src/views/count.vue:136]: Text is localized.

**Pull request merged**